app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([
+    'underscore',
+    'jquery',
+    'Magento_Ui/js/grid/columns/column'
+], function (_, $, Column) {
+    'use strict';
+
+    return Column.extend({
+        defaults: {
+            previewImageSelector: '[data-image-preview]',
+            visibile: null,
```

Comment:
Small typo - should be `visible`. I searched around and this time I don't think that's a typo in the class we're inheriting from 😅

Reply:
Changed to `visibleRecord`. `visible` is used by parent for a different purpose.